cmd/xmidt-agent/main_test.go

```diff
@@ -141,93 +137,6 @@ func Test_xmidtAgent(t *testing.T) {
 	}
 }
 
-type badShutdown struct{}
-
-func (*badShutdown) Shutdown(...fx.ShutdownOption) error {
-	return errors.New("random shutdown error")
-}
-
-func Test_xmidtAgent_lifecycle(t *testing.T) {
```

Comment:
FYI we're getting rid of our centralize lifecycle functions in main, so we wont need these tests